[UXIT-1706] Enhance Clarity and Consistency in ‘What is Filecoin’ Section

mirhamasala opened the pull request:
This PR improves readability and consistency in multiple sections of the Filecoin documentation, including updates to language flow, terminology alignment, and clearer explanations in key areas such as the storage model, retrieval market, blockchain, crypto-economics, networks, and programming on Filecoin.

smagdali reviewed (changes requested):
This is great. I'm definitely not a fan of American-style 'Capitalising All Headings No Matter What' though, and that's not what we do elsewhere on the site? 

Huh, the readme.md on github used to have a style guide in it, I guess Johnny deleted it when we moved to Gitbook.

smagdali commented on basics/what-is-filecoin/blockchain.md:
```diff
@@ -1,119 +1,111 @@
-* Verified Registry Actor - responsible for managing verified clients.
-* Ethereum address Manager (EAM) Actor- responsible for assigning all Ethereum compatible addresses on Filecoin Network, including EVM smart contract addresses and Ethereum account addresses.
-* EVM Account Actor - a non-singleton built-in actor representing an external Ethereum identity backed by a secp256k1 key.
+- **System Actor**: General system actor.
```
These should all be normal case. "Init actor" - they are not proper nouns (and yes, this is the only one that was correct before)

mirhamasala replied:
Okay - done.

smagdali commented on networks/calibration/explorers.md:
```diff
@@ -12,7 +12,7 @@ description: >-
 ## FILFOX
 
-[https://calibration.filfox.info/en](https://calibration.filfox.info/en)
+[https://calibration.filfox.info/en](http://47.109.105.51/en)
```
this situation is temporary - I'm trying to figure out what's going on with them but we should probably list both for now.

mirhamasala replied:
It doesn't pass the link checker test if the link is broken and thus the PR can't be merged.. We can revert is once they fix it?

mirhamasala commented:
@smagdali - I quite like the visual hierarchy that capitalising heading provides. And was planning for us to implement it throughout the docs. Following your comment, I reverted it.

> This is great. I'm definitely not a fan of American-style 'Capitalising All Headings No Matter What' though, and that's not what we do elsewhere on the site?

> Huh, the readme.md on github used to have a style guide in it, I guess Johnny deleted it when we moved to Gitbook.

Yes, a style guide would be helpful.
